packages/valaxy-theme-yun/components/third/YunTwikoo.vue

```diff
@@ -2,9 +2,14 @@
 import * as addonTwikoo from 'valaxy-addon-twikoo'
 import 'valaxy-addon-twikoo/client/styles/index.scss'
 import { isEmptyAddon } from 'valaxy'
+import { onBeforeUnmount } from 'vue'
 
 if (!isEmptyAddon(addonTwikoo))
   addonTwikoo.useTwikooWithOptions()
+
+onBeforeUnmount(() => {
+  document.querySelector('#twikoo')?.remove()
```

Comment:
Why we need this?

Reply:
twikoo will replace `<div id="tcomment" w="full" />` to `<div id="twikoo" class="twikoo"></div>`. When component unmount, `<div id="twikoo" class="twikoo"></div>` isn't removed correctly.